packages/itwinui-react/src/core/Table/Table.tsx

```diff
@@ -219,7 +219,7 @@ export type TableProps<
    * Modify the density of the table (adjusts the row height).
    * @default 'default'
    */
-  density?: 'default' | 'condensed' | 'extra-condensed';
+  density?: 'default' | 'condensed' | 'extra-condensed' | 'mega-condensed';
```

Comment:
I vote `most-condensed` and we make the table as condensed as possible so there can't be an even more condensed version.

In [our next major release](https://github.com/iTwin/iTwinUI/milestone/2) we can consider setting the size via a data attribute and use consistent size naming as other components (small, medium, large maybe).

Reply:
Oops yea let me clean that up